EngineDriver/src/main/java/jmri/enginedriver/threaded_application.java

```diff
@@ -159,8 +159,8 @@
 
     String client_address; //address string of the client address
     Inet4Address client_address_inet4; //inet4 value of the client address
-    String client_ssid = "UNKNOWN";    //string of the connected SSID
-    String client_type = "UNKNOWN"; //network type, usually WIFI or MOBILE
+    String client_ssid;    //string of the connected SSID
```

Comment:
@flash62au Peter, did you mean to remove the "UNKNOWN" values? I added them to prevent an NPE.